### PR TITLE
New stfs2s3 tables

### PR DESCRIPTION
**Pull request by mark-walle**
@danpollock can you please review this?

### CODE REVIEW

**Comment by mark-walle** [comment]
@BJenning adding you as a reviewer.

File: microservices/s3_to_redshift/ddl/servicebc.call_centre_esb_data.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS servicebc.call_centre_esb_data (
+  "ID" DECIMAL(13,0) NOT NULL PRIMARY KEY ENCODE ZSTD,
+  "SkillGroup" VARCHAR(50) ENCODE ZSTD,
+  "ContactReason" VARCHAR(50) ENCODE ZSTD,
+  "ContactReasonDetail" VARCHAR(50) ENCODE ZSTD,
+  "DateKey" INTEGER ENCODE ZSTD,
+  "INTEGERerval" VARCHAR(5) ENCODE ZSTD,
```
Comment (mark-walle) [inline microservices/s3_to_redshift/ddl/servicebc.call_centre_esb_data.sql]: @BJenning this looks like what you're referring to. My mistake due to a Find and Replace. I'll check that the table isn't actually this, and fix it if it is. I'll fix the code now.

Comment (mark-walle) [reply microservices/s3_to_redshift/ddl/servicebc.call_centre_esb_data.sql]: The table is indeed also incorrect. Fixing.

File: microservices/s3_to_redshift/ddl/servicebc.call_centre_esb_data.sql [hunk 2]
```diff
@@ -4,7 +4,7 @@ CREATE TABLE IF NOT EXISTS servicebc.call_centre_esb_data (
   "ContactReason" VARCHAR(50) ENCODE ZSTD,
   "ContactReasonDetail" VARCHAR(50) ENCODE ZSTD,
   "DateKey" INTEGER ENCODE ZSTD,
-  "INTEGERerval" VARCHAR(5) ENCODE ZSTD,
+  "Interval" VARCHAR(5) ENCODE ZSTD,
```
Comment (mark-walle) [inline microservices/s3_to_redshift/ddl/servicebc.call_centre_esb_data.sql]: @BJenning fixed here.

Comment (BJenning) [reply microservices/s3_to_redshift/ddl/servicebc.call_centre_esb_data.sql]: Ok. Thanks.

**Comment by mark-walle** [comment]
@danpollock you're still a reviewer on this. Can I remove you, or do you still want to review before I merge the PR based on @BJenning's approval?

**Comment by mark-walle** [comment]
removing @danpollock as a reviewer. Merging now.

**Pull request merged**